### DISCUSSION TITLE
Tutorial Manual - Slope failure under transient hydraulic loads Example

### DISCUSSION THREAD

**Discussion by ema5858** [Anura3D/Anura3D_OpenSource#176] [Q&A]
Dear All,

I have been practicing "Slope failure under transient hydraulic loads" example by following the Tutorial Manual. However, in the calculation process, I encountered the error as follows:
forrtl: severe (59): list-directed I/O syntax error

I checked all the inputs, the only thing I am not sure is the file with HHBF extension. In the Tutorial Manuel, it was written to create this file for river level. The name of the file is the same as the GOM and CPS_001 files, as it is required, but the files I attached here are just the copy of them in text files. I also put my files and the screenshot of the encountered error in this post. Will you please help me with this ? Thank you in advance.

Best regards

<img width="861" alt="Screenshot 2023-12-22 22
[chapter12 - CopyGOM.txt](https://github.com/Anura3D/Anura3D_Open
[CPS_001.txt](https://github.com/Anura3D/Anura3D_OpenSource/files/13757859/CPS_001.txt)
Source/files/13757858/chapter12.-.CopyGOM.txt)
2015" src="https://github.com/Anura3D/Anura3D_OpenSource/assets/154570946/be8d4691-c470-499c-9e0a-2470ca383870">

**Comment by FrancescaCeccato** [comment]
Hi!
Actually the error message identify an error in the GOM file, are you sure that all input are correct?

In your case the file for river level should be named chapter12.HHBF, I attach an example of file (change extension).
[ExampleHHBF.txt](https://github.com/Anura3D/Anura3D_OpenSource/files/13816957/ExampleHHBF.txt)

**Comment by ema5858** [comment]
Dear Dr Caccato, 

Thank you for your response. Yes, I checked and both my file name and the input values are the same.

**Comment by FrancescaCeccato** [comment]
[RapidDrawDown.zip](https://github.com/Anura3D/Anura3D_OpenSource/files/13857500/RapidDrawDown.zip)
You can check these input filoe that used to work.
